yarn.lock

```diff
@@ -1981,6 +1981,15 @@
     lodash "^4.5.0"
     prop-types "^15.6.1"
 
+"@unimodules/react-native-adapter@~5.0.0-alpha.0":
```

Comment:
These modules should be linked and not in the yarn.lock file

Reply:
yeah, I think sth is messed up again, that's what I get after rebasing with master and running yarn in ncl